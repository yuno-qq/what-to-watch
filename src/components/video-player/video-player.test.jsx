import React from 'react'
import {describe, test, expect} from "@jest/globals"
import renderer from 'react-test-renderer'

import VideoPlayer from "./video-player.jsx"


describe(`<VideoPlayer> snapshots:`, () => {
  test(`should render "Черное воскресенье"`, () => {
    const videoData = {
      imageSrc: `/black-sunday-image`,
      videoSrc: `/black-sunday-preview`,
    }

    const tree = renderer.create(<VideoPlayer
      isLoading={true}
      isPlaying={false}
      setIsLoading={() => {}}
      imageSrc={videoData.imageSrc}
      videoSrc={videoData.videoSrc}
    />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
