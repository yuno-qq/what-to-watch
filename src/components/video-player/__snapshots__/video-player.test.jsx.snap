// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer> snapshots: should render "Черное воскресенье" 1`] = `
<video
  className=""
  loop={true}
  muted={true}
  onCanPlay={[Function]}
  onLoadedMetadata={[Function]}
  onPause={[Function]}
  onPlay={[Function]}
  onTimeUpdate={[Function]}
  onWaiting={[Function]}
  poster="/black-sunday-image"
  src="/black-sunday-preview"
/>
`;

exports[`<VideoPlayer> snapshots: should render "Черное воскресенье" with className "video-player" 1`] = `
<video
  className="video-player"
  loop={true}
  muted={true}
  onCanPlay={[Function]}
  onLoadedMetadata={[Function]}
  onPause={[Function]}
  onPlay={[Function]}
  onTimeUpdate={[Function]}
  onWaiting={[Function]}
  poster="/black-sunday-image"
  src="/black-sunday-preview"
/>
`;
